soc/arm/nxp_kinetis/ks2x/dts_fixup.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
This file has been phased out, you should be able to remove this.

Reply:
done